### ISSUE TITLE
CMake complains about wrong missing package

### ISSUE THREAD

**Issue by mpeter50** [rncbc/qjackctl#174]
After the repository has been cloned and the `qjackctl_0_9_6` tag has been checked out, building the project with `cmake -B build` results in an error message complaining about the `jack` package missing. This is on openSUSE Leap 15.3.

I was scratching my head for some time because the `jack` package is installed, but then I went on to search in the issues and found [this comment](https://github.com/rncbc/qjackctl/issues/75#issuecomment-543313419) saying that the JACK development packages are needed too.
After installing the `libjack-dev` package with zypper I was able to build, install and run QjackCtl.

Would it be possible to make it more clear in the error message which packages are needed?
I'm not familiar with CMake, so this might be something internal, though.

Also, I didn't want to an open a separate issue for the following until I know whether it belongs to this issue tracker.
Do you have information on why openSUSE Leap 15.3 has only 0.4.3 as the latest version in their repositories?

**Comment by rncbc** [comment]
no, i realy don't know (i'm no internal to opensuse for that matter whatsoever:))

and on rpm distros like opensuse, mind you, you actually need the **-devel** packages (not the debianese -dev;))

cheers

**Comment by mpeter50** [comment]
Oh, yes, I messed that up :D fixed it in the initial comment.
